src/main/java/kr/sesac/aoao/server/dino/controller/DinoController.java

```diff
@@ -30,4 +31,40 @@ public ResponseEntity<GetUserDinoResponse> getDinoInfo(@PathVariable Long userId
 		GetUserDinoResponse userDinoResponse = dinoService.getDinoInfo(userId);
 		return ResponseEntity.ok(userDinoResponse);
 	}
+
+	/**
+	 * 다이노 이름 변경
+	 * @since 2024.01.22
+	 * @return GetUserDinoResponse
+	 * @author 김은서
+	 */
+	@PostMapping("/dinos/rename")
+	public ResponseEntity<GetUserDinoResponse> renameDino(Long dinoId, String name){
+		GetUserDinoResponse userDinoResponse = dinoService.renameDino(dinoId, name);
+		return ResponseEntity.ok(userDinoResponse);
+	}
```

Comment:
```suggestion
	public ResponseEntity<Application<GetUserDinoResponse>> renameDino(Long dinoId, String name){
		GetUserDinoResponse userDinoResponse = dinoService.renameDino(dinoId, name);
		return ResponseEntity.ok(ApplicationResponse.success(userDinoResponse));
	}
```

요런 식으로 공통 response객체를 사용한다면 모든 API 에서의 응답 형식을 통일화 시킬 수 있습니다!